integration/test_devices.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+
+def test_block(connection):
+    response = connection.devices.block(251691)
+    assert response.status_code == 201
+
+
+@pytest.mark.skip("Failing with 404.")
+def test_get_agent_full_disk_access_state(connection):
+    response = connection.devices.get_agent_full_disk_access_state(891804948033320117)
```

Comment:
The newer IDs (the longer ones) should probably be passes in as strings. That is how they get returned on responses.

The legacy IDs are numbers though.

Reply:
As discussed, skipping this tests.